app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedStoryClickListener.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.android.app.home.recentlyplayed
+
+import org.oppia.android.app.model.PromotedStory
+
+/** Listener interface for when promoted story is clicked in the UI. */
+interface PromotedStoryClickListener {
+  fun promotedStoryClicked(promotedStory: PromotedStory)
```

Comment:
Could we go ahead and add  KDoc here? It'll help us avoid needing to carry over the KDoc exemption.

Reply:
> Thanks @adhiamboperes just had a few comments.
> 
> Regarding the layouts, when is ongoing_story_card vs. promoted_story_card used? It'd be nice to fix this last case, too, if it's the only other occurrence of "ongoing story" left.

Note: `ongoing_story_card`  is used to show stories in the "recently played screen", under "Played within the last week and recommended stories", shown when a user clicks "View all promoted stories" from the home screen. I have renamed it to `recently_played_story_card.xml`.

`promoted_story_card.xml` is the layout file used to show promoted stories on the home page.

Reply:
I've added kdoc for the interface method